app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :author, class_name: 'User', foreign_key: :author_id
-  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+  after_save :update_likes_counter
+
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  private
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.length)
```

Comment:
- **[OPTIONAL]** Following the feedback earlier I think we can modify this as well.

  ```
  def update_likes_counter
    post.increment!(:likes_counter)
  end
  ```